gitdm/models.py

```
@@ -74,28 +83,98 @@
+            return self.user.get_full_name() or self.user.email
+        return "Patient"
+
+    @property
+    def age(self):
+        if self.dob:
+            today = timezone.now().date()
+            return today.year - self.dob.year - ((today.month, today.day) < (self.dob.month, self.dob.day))
+        return None
+
+    def clean(self):
+        # Patient’s user must actually be a patient
```

Ruff (0.12.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)